java/runtime/src/main/java/org/ray/runtime/object/ObjectStore.java

```diff
@@ -69,8 +69,8 @@ public void put(Object object, ObjectId objectId) {
           "Trying to put a NativeRayObject. Please use putRaw instead.");
     }
     NativeRayObject serialized = ObjectSerializer.serialize(object);
-    List<ObjectId> innerObjectIds = RayObjectSerializer.getAndClearContainedObjectIds();
-    putRaw(serialized, objectId, innerObjectIds);
+    serialized.setContainedObjectIds(RayObjectImpl.getAndClearContainedObjectIds());
```

Comment:
Please put the `serialized.setContainedObjectIds(RayObjectImpl.getAndClearContainedObjectIds())` into `ObjectSerializer.serialize(object)`. So we don't need to remember to call it after every `ObjectSerializer.serialize()` call.

Reply:
👍